forum/migration_helpers.py

```diff
@@ -139,10 +139,14 @@ def create_or_update_comment(comment_data: dict[str, Any]) -> None:
             child_count=comment_data.get("child_count", 0),
             created_at=make_aware(comment_data["created_at"]),
             updated_at=make_aware(comment_data["updated_at"]),
+            depth=1 if parent else 0,
         )
         mongo_comment.content_object_id = comment.pk
         mongo_comment.content_type = comment.content_type
         mongo_comment.save()
+        sort_key = f"{parent.pk}-{comment.pk}" if parent else f"{comment.pk}"
+        comment.sort_key = sort_key
+        comment.save()
```

Comment:
Is there a reason why we are calling `comment.save()` here, and not adding `sort_key` to `Comment.objects.create(...)`?

Reply:
yeah `sort_key` needs an self.id/self.pk of the comment to make.